### DISCUSSION TITLE
Thoughts about AI

### DISCUSSION THREAD

**Discussion by kryoqa** [shaaanuu/endernote#7] [Ideas]
I have been following the app's development for some time. I know it is still a WIP. 

The app's direction is good. There isn't a proper obsidian replacement in foss (logseq - different workflow, trilium - no native android app, joplin - editing on phones are not good) I do hope this app bridges the gap and provides a proper replacment. 

I have a request. I really **_dislike AI_**. I would prefer if the app doesn't have AI integrated like **every** other foss app has now. If you do plan on adding AI features to the app, kindly make it a separate plugin/extension that users have to enable. I would prefer there it isn't built into the app. Obsidian's main advanatage is encryption hence privacy. Having AI just reduces that imo.

I've seen many foss apps chasr the AI trend. Some are so badly implemented that they are difficult to remove. For users who want a native AI experience, there are many foss apps - siyuan, reor, appflowy.

**Comment by shaaanuu** [comment]
Well don't worry, I am one of the haters of the Ai hype😂
